.babelrc-optimize.js

```diff
@@ -0,0 +1,15 @@
+// See `wiki/consuming.md` for rationale and consumer requirements.
+
+const baseConfig = require('./.babelrc.js');
+// Skip `propType` generation
+baseConfig.plugins.splice(
+  baseConfig.plugins.indexOf(
+    './scripts/babel/proptypes-from-ts-props'
+  ),
+  1
+);
+// Transform runtimes using babel plugin.
+// Requires consming applications to use `@babel/runtime`.
```

Comment:
it me, [the typo hunter](https://i.redd.it/84z2r6ty3h671.jpg)
```suggestion
// Requires consuming applications to use `@babel/runtime`.
```

Reply:
Also wanted to add I think `@babel/runtime` is a worthwhile dependency to add - I think I vaguely remember the previous component library I've worked on having it as well, and it fits alongside typescript, prop-types, etc.

Just to clarify, would we straight up add this to our `peerDependency` list in `package.json` (+ accompanying Getting Started documentation) or is this requirement only formally in notes about `optimize/`?

Reply:
> Just to clarify, would we straight up add this to our peerDependency list in package.json (+ accompanying Getting Started documentation) or is this requirement only formally in notes about optimize/?

For now I'm ok simply noting it in comments and the wiki. If we want to formalize it, I think we begin requiring it for both dev and prod builds. At that point we'd probably add it to `peerDependencies` (although I haven't seen great guidance on whether this is the correct method).

Reply:
OK, makes sense! I'm ++ for formalizing it later for dev/prod 🎉